app/experimenter/nimbus-ui/src/components/PageEditOverview/index.test.tsx

```diff
@@ -48,23 +49,34 @@ describe("PageEditOverview", () => {
   };
 
   beforeEach(() => {
-    mockSubmitData = {
+    mockMutationData = {
       changelogMessage: CHANGELOG_MESSAGES.UPDATED_OVERVIEW,
       name: experiment.name,
       hypothesis: experiment.hypothesis!,
       publicDescription: experiment.publicDescription!,
       riskMitigationLink: experiment.riskMitigationLink!,
+      riskBrand: experiment.riskBrand!,
+      riskRevenue: experiment.riskRevenue!,
+      riskPartnerRelated: experiment.riskPartnerRelated!,
+    };
+    mockSubmitData = {
+      ...mockMutationData,
+      ...{
+        riskBrand: String(experiment.riskBrand!),
+        riskRevenue: String(experiment.riskRevenue!),
+        riskPartnerRelated: String(experiment.riskPartnerRelated!),
+      },
```

Comment:
This kinda sucks, but basically we submit a stringified boolean, and the GQL mutation needs to mock a real boolean.

Reply:
ISTR we also run into this in a few spots where we deal with a number as a string but then need to deal with it explicitly as a parsed number in GQL mocks. 🤔

Reply:
@lmorchard I think I fixed all the string ints for ids already.  Why are we treating bools as strings?

Reply:
The form radio buttons produce strings, but the server wants booleans. So in out tests we need to (well, we should to be accurate) mock submitting the form with strings and then convert them to bools once they hit the mutation